src/processors/enrichment.py

```
@@ -392,15 +394,53 @@ def _format_template_analysis(template: str, analysis: dict) -> str:
     return "\n".join(lines)
 
 
+def _build_enrichment_markdown(
+    job: dict, enrichment: Enrichment, template_analysis: dict | None
+) -> str:
+    """Drive doc content for the enrichment result — uploaded as {job_id}_enriched_long.md (ADR-0057)."""
+    ts = datetime.now(timezone.utc).isoformat()
```

Ruff (0.16.3)

[warning] Use `datetime.UTC` alias

Convert to `datetime.UTC` alias

(UP017)